### DISCUSSION TITLE
put a " , " after every list item, but not the last one

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1487] [Help] [answered]
Dear all,
I am using the following code, which works pretty well. The only thing is the "," after {{name}}. The intention is to separate the tags, but the "," shall not show up after the last tag name.

Example: the code lists like: tag1, tag2, tag3,
What I want: tag1, tag2, tag3

Any idea how to achieve this?

```
{#checkIfAny @config.post.displayTags @config.post.displayShareButtons }}
                     {{#if @config.post.displayTags}}
                        {{#if tags}}
                            {{#each tags}}
                                <a href="{{url}}">{{name}},
                                </a>
                            {{/each}}
                        {{/if}}
                     {{/if}}
                     {{/checkIfAny}}

**Comment by sandrobilbeisi** [comment]
for starters, you should not include the comma inside the link

correct syntax:
`<a href="{{url}}">{{name}}</a>,`

**Reply by 3muchfun** [reply]
oh my

**Comment by bobmitro** [comment, marked as answer]
@3muchfun Try this:

```
{{#if tags}}
   {{#each tags}}{{#checkIf @index '>' 0}}, {{/checkIf}}<a href="{{url}}">{{name}}</a>{{/each}}
{{/if}}
```

**Reply by 3muchfun** [reply]
That puts the " , " at the right places, Thank you, @bobmitro
